editor/src/components/inspector/common/inspector-end-to-end-tests.spec.browser.tsx

```diff
@@ -87,15 +87,15 @@ describe('inspector tests with real metadata', () => {
       leftControl.attributes.getNamedItemNS(null, 'data-controlstatus')?.value,
     ).toMatchInlineSnapshot(`"simple"`)
 
-    expect(bottomControl.value).toMatchInlineSnapshot(`""`)
+    expect(bottomControl.value).toMatchInlineSnapshot(`"178"`)
```

Comment:
this is interesting. why does the bottomControl changed from empty to 178? is it because the `detected` value was 178px and the `px` part screwed with the parsing and the logic fall back to unset?

Reply:
yes, now all layout controls show computedStyle with light grey color. this is good because now it is matching how other controls like minwidth/padding/margin works.